eng/helix/configure-helix-machine.ps1

```diff
@@ -0,0 +1,18 @@
+# This file prepares the helix machine for our tests runs
+
+$dotnetLocation = Join-Path (Split-Path -Parent $script:MyInvocation.MyCommand.Path) "dotnet"
+
+Write-Output "Adding dotnet location to path: $dotnetLocation"
+
+# Prepend the path to ensure that dotnet.exe is called from there.
+$env:path="$dotnetLocation;$env:path"
+
+# Don't look outside the TestHost to resolve best match SDKs.
+$env:DOTNET_MULTILEVEL_LOOKUP = 0
+
+# Disable first run since we do not need all ASP.NET packages restored.
```

Comment:
Why disable this? 

By design, all 3 framework packages are always restored. We should retain whatever customer experiences we plan on shipping.

Reply:
that's fine, the arcade script does this but it probably isn't necessary